### PR TITLE
Worker packages version update

### PR DESCRIPTION
**Pull request by surgupta-msft**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Grpc</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>10</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
-    <VersionSuffix></VersionSuffix>
+    <PatchProductVersion>2</PatchProductVersion>
```
Comment (liliankasem) [inline src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj:12]: Patch: 0

File: src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj [hunk 2]
```diff
@@ -9,8 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Grpc</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>10</MinorProductVersion>
```
Comment (liliankasem) [inline src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj:11]: Minor: 11

File: src/DotNetWorker.Core/DotNetWorker.Core.csproj [hunk 3]
```diff
@@ -9,7 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Core</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>12</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
+    <PatchProductVersion>2</PatchProductVersion>
```
Comment (liliankasem) [inline src/DotNetWorker.Core/DotNetWorker.Core.csproj:12]: Patch 0 (or remove)

File: src/DotNetWorker.Core/DotNetWorker.Core.csproj [hunk 4]
```diff
@@ -9,7 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Core</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>12</MinorProductVersion>
```
Comment (liliankasem) [inline src/DotNetWorker.Core/DotNetWorker.Core.csproj:11]: Minor 13

File: sdk/Sdk/Sdk.csproj [hunk 5]
```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <MinorProductVersion>10</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
+    <MinorProductVersion>11</MinorProductVersion>
+    <PatchProductVersion>0</PatchProductVersion>
```
Comment (liliankasem) [inline sdk/Sdk/Sdk.csproj:5]: Can probably remove this (PatchProductVersion)

Comment (liliankasem) [reply sdk/Sdk/Sdk.csproj:5]: Or keep it empty but should be consistent with all csproj files

File: src/DotNetWorker/DotNetWorker.csproj [hunk 6]
```diff
@@ -8,9 +8,9 @@
     <AssemblyName>Microsoft.Azure.Functions.Worker</AssemblyName>
     <RootNamespace>Microsoft.Azure.Functions.Worker</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <MinorProductVersion>14</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
-    <VersionSuffix></VersionSuffix>
+    <MinorProductVersion>13</MinorProductVersion>
```
Comment (liliankasem) [inline src/DotNetWorker/DotNetWorker.csproj:11]: 15?

File: extensions/Worker.Extensions.CosmosDB/src/Worker.Extensions.CosmosDB.csproj [hunk 7]
```diff
@@ -7,7 +7,7 @@
 
     <!--Version information-->
     <VersionPrefix>4.1.0</VersionPrefix>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
```
Comment (liliankasem) [inline extensions/Worker.Extensions.CosmosDB/src/Worker.Extensions.CosmosDB.csproj:10]: Just realised we bumped this to 4.2.0

https://github.com/Azure/azure-functions-dotnet-worker/blob/0d49e191d454669fde6186559404df1e2ac1c107/extensions/Worker.Extensions.CosmosDB/src/Properties/AssemblyInfo.cs#LL7C81-L7C86

so this should be 4.2.0-preview1

File: extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj [hunk 8]
```diff
@@ -7,7 +7,7 @@
 
     <!--Version information-->
     <VersionPrefix>5.1.0</VersionPrefix>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
```
Comment (liliankasem) [inline extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj:10]: I think we should bump beta to stable release 5.1.0 or 5.1.1

https://github.com/Azure/azure-functions-dotnet-worker/blob/0d49e191d454669fde6186559404df1e2ac1c107/extensions/Worker.Extensions.Storage.Blobs/src/Properties/AssemblyInfo.cs#LL7C34-L7C82

[5.1.2](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/5.1.2)

Comment (surgupta-msft) [reply extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj:10]: Do you mean, we don't need this in preview now? so 5.1.2 should be the version?

Comment (liliankasem) [reply extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj:10]: No our blob worker ext should stay preview, but we should at least update the assembly info to not point to 5.1.0-beta

**Review by liliankasem — APPROVED** [review]

**Pull request merged**